modules/distribution/src/main/config/log4j2.properties

```diff
@@ -0,0 +1,37 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+# agreements.  See the NOTICE file distributed with this work for additional information regarding
+# copyright ownership.  The ASF licenses this file to you under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with the License.  You may
+# obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License
+# is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+# or implied. See the License for the specific language governing permissions and limitations under
+# the License.
```

Comment:
The formatting of this license header seemed to get mangled a bit. Could be a subsequent fix, to use license-maven-plugin like Accumulo does.

Reply:
I am copied this file from Accumulo and modified it.  I had the license from Accumulo and for some reason the rat plugin here in Fluo did not like it, so I used the one from the old Fluo log4j props file.  I was curious about what was going on, but didn't have time to look into it.

Reply:
It's because the old rat didn't allow https URLs. Updating to at least rat 0.14 will fix that.